airflow/providers/amazon/aws/operators/base_aws.py

```diff
@@ -85,10 +86,12 @@ def __init__(
         region_name: str | None = None,
         verify: bool | str | None = None,
         botocore_config: dict | None = None,
+        region: str | None | ArgNotSet = NOTSET,  # Required for `.partial` signature check
         **kwargs,
     ):
+        additional_params = {} if region is NOTSET else {"region": region}
         hook_params = AwsHookParams.from_constructor(
-            aws_conn_id, region_name, verify, botocore_config, additional_params=kwargs
+            aws_conn_id, region_name, verify, botocore_config, additional_params=additional_params
```

Comment:
Is not it a breaking change? `kwargs` is no longer passed to `from_constructor`

Reply:
Nope... `from_constructor` it is internal method of internal class

Reply:
The only one purpose it is create frozen dataclass in constructor and resolve `region_name` and `region` in AwsBaseOperator / AwsBaseSensor, the object do not exposed anywhere